.gitignore

```diff
@@ -80,4 +80,4 @@ generated/
 .firebase
 
 # Ruby bundler files for fastlane
-.bundle
\ No newline at end of file
+.bundle
```

Comment:
Let's stick to newlines at the end of files.

Reply:
Totally agree😇